haystack/nodes/prompt/invocation_layer/hugging_face.py

```diff
@@ -136,9 +147,18 @@ def invoke(self, *args, **kwargs):
                     "return_full_text",
                     "clean_up_tokenization_spaces",
                     "truncation",
+                    "generation_kwargs",
                 ]
                 if key in kwargs
             }
+            generation_kwargs = model_input_kwargs.pop("generation_kwargs", self.generation_kwargs)
+            if isinstance(generation_kwargs, dict):
+                model_input_kwargs.update(generation_kwargs)
+            elif isinstance(generation_kwargs, GenerationConfig):
+                gen_dict = generation_kwargs.to_diff_dict()
+                gen_dict.pop("transformers_version", None)
```

Comment:
I found this one to be included, and it needs to be removed because the underlying pipeline doesn't know about this parameter and raises an exception.